nodeJS/express/controllers.md

```diff
@@ -326,113 +373,138 @@ Also, as we've discussed earlier with regards to calling the `next` function. We
 1. With the string `next('route')` - Will pass control to the next route handler with the same matching path (if there is one). This only works for `app.METHOD` or `router.METHOD`. Potentially, it can also be the same as just calling `next` with no argument.
 1. With the string `next('router')` - Will skip all middleware functions attached to the specific router instance and pass control back out of the router instance. Basically, we exit the router and go back to the parent router, e.g. `app` (yes, the Express app is also just a router under the hood).
 
-Out of the four, you will likely only use the first two, unless you have a very specific need that requires the other two.
+Out of the four, we will likely only use the first two, unless we have a very specific need that requires the other two.
 
 ### Consolidating what we've learned
 
-The following is the folder structure you'd end up with if you have been coding along:
+The following is the folder structure we'd end up with if you have been coding along:
 
 ```text
 express-app/
+├─ errors/
+│  ├─ CustomNotFoundError.js
 ├─ controllers/
-│  ├─ userController.js
+│  ├─ authorController.js
 ├─ routes/
-│  ├─ userRoutes.js
+│  ├─ authorRouter.js
+│  ├─ ... other routers
 ├─ app.js
+├─ db.js
 ```
 
-Creating a user controller file that contains well-defined responsibilities to handle "User".
+Creating a controller file that contains well-defined responsibilities to handle "Author".
```

Comment:
On reflection, I don't actually think we need the file recap from here to line 503. I think it's enough to just share the file structure diagram and that's it. You've already improved the explicitness of codealong instructions earlier in the lesson (inc. the changes for the Routes lesson). I don't think the comments in the code blocks are vital for the lesson, and if you feel any are, you probably should bring them into the main lesson content up above as appropriate.

But IMHO, we can just go straight to "As an exercise, feel free to create more sample data in our `db.js` file and controllers for the other routes we've previously created." after the file structure diagram.

Reply:
you're right it's very unconventional as well in the curriculum, will remove it